includes/Classifai/Providers/Provider.php

```diff
@@ -397,6 +430,44 @@ public function render_auto_caption_fields( $args ) {
 		}
 	}
 
+	/**
+	 * Render allowed users table.
+	 *
+	 * @param array $args The args passed to add_settings_field
+	 */
+	public function render_allowed_users( array $args = array() ) {
+		$setting_index = $this->get_settings();
+		$value         = ( isset( $setting_index[ $args['label_for'] ] ) ) ? $setting_index[ $args['label_for'] ] : array();
```

Comment:
This can be simplified I think:

```suggestion
		$value         = $setting_index[ $args['label_for'] ] ?? array();
```